tests/data/modules/test_module_09.f

```diff
@@ -30,7 +30,7 @@ MODULE MYMOD9
 
       REAL, PARAMETER :: 
      &    PI = 3.14159265,
-     &    RAD=PI/180.0
+     &    RAD=0.0174532925
```

Comment:
@pratikbhd Please restore this test to the original form.  I'm not too concerned that the translator does not currently handle expressions on the RHS of a PARAMETER assignment, causing the original version of this test to fail; but I want the fact that this feature is currently not handled to be recorded somewhere so that we remember to come back and address it.  I think an easy way to remember this is to have a test input that captures the language feature that needs to be handled.

Reply:
Test program for this construct has been in `tests/data/cases` directory

Reply:
Thanks.  Resolved.